source/brailleDisplayDrivers/papenmeier_serial.py

```diff
@@ -9,15 +9,14 @@
 

```

Comment:
Could you find out whether this driver is still in use by some people? I think that it is very unlikely that this will ever be tested before a release, due to the age of these devices.